### DISCUSSION TITLE
Windows 10 Ransomware Protection

### DISCUSSION THREAD

**Discussion by chrisjlocke** [sqlitebrowser/sqlitebrowser#2713] [General]
Seems there is a crazy setting in Windows 10 that prevents 'unauthorised' applications creating files on the desktop or in the documents folder.

https://user-images.githubusercontent.com/7872218/119094839-71b39a80-ba09-11eb-8ffa-f159ce5711ca.mp4

This shows as DB4S unable to create a new database.  Windows does display a notification explaining why, but then hides it, so some users could moan and blame DB4S.

Annoyingly, go into 'Ransomware Protection' and it says 'no action needed'.  You then have to dig through various screens to get to the action to allow it.

https://user-images.githubusercontent.com/7872218/119095313-09b18400-ba0a-11eb-9b66-b118d9b8df15.mp4

I'm sure it'll catch some people out....

**Comment by scottfurry** [comment]
Some would have thought the PTB's in Redmond would have learned from this mistake...12 years and 4 OS's later...
[Mac Vs PC commercials - Cancel or Allow?](https://www.youtube.com/watch?v=8CwoluNRSSc)
